addons/mail/static/src/core/common/discuss_model.js

```diff
@@ -0,0 +1,15 @@
+/* @odoo-module */
+
+import { toRaw } from "@odoo/owl";
+
+export class DiscussModel {
+    /** @param {DiscussModel} record */
+    eq(record) {
+        return toRaw(this) === toRaw(record);
```

Comment:
Just to make sure, and I guess it's fine if tests are passing, but is `toRaw` properly observed? That is: if both parts of `eq` used alone in a template would trigger rerender if they change?

Reply:
Pretty sure this doesn't affect owl's reactivity, similarly to `rec1 === rec2` doesn't by itself.
`props.rec1 === store.records[localId2]` does thanks to accessors `.rec1` / `.records` / `[localId2]`, not thanks to `===` expression.

"Observing" from owl's reactivity consists of a read on the proxy of object. `toRaw()` returns a raw version, so `toRaw().x` won't register reactive read on `x`. Inner-code of this function doesn't need reactivity.